src/Tribe/Main.php

```diff
@@ -237,12 +237,67 @@ protected function __construct() {
 			$this->pluginDir  = $this->plugin_dir = trailingslashit( basename( $this->plugin_path ) );
 			$this->pluginUrl  = $this->plugin_url = plugins_url( $this->plugin_dir );
 
+			// Set common lib information, needs to happen file load
 			$this->maybe_set_common_lib_info();
 
 			// let's initialize tec silly-early to avoid fatals with upgrades from 3.x to 4.x
 			add_action( 'plugins_loaded', array( $this, 'plugins_loaded' ), 0 );
 		}
 
+		/**
+		 * To avoid duplication of our own methods and to provide a underlying system
+		 * Modern Tribe maintains a Library called Common to store a base for our plugins
+		 *
+		 * Currently we will read the File `common/package.json` to determine which version
+		 * of the Common Lib we will pass to the Auto-Loader of PHP.
+		 *
+		 * In the past we used to parse `common/src/Tribe/Main.php` for the Common Lib version.
+		 *
+		 * @link https://github.com/moderntribe/tribe-common
+		 * @see  self::init_autoloading
+		 *
+		 * @return void
+		 */
+		public function maybe_set_common_lib_info() {
+			// Check if common has a package.json file, if not just bail
+			$common_package_file = $this->plugin_path . 'common/package.json';
```

Comment:
We don't include`package.json` in any of our packaged zips, so this won't fly. That's why we were parsing `Main.php`.